### ISSUE TITLE
aws_emr_cluster instance_group not recognized

### ISSUE THREAD

**Issue by kh360** [terraform-providers/terraform-provider-aws#1716]
Unable to configure `aws_emr_cluster` using the new `instance_group` syntax. Upgraded from `0.9.8` yesterday to have access to the new EMR features, so it could be operator error or local configuration error.

```
$ terraform -v
Terraform v0.10.6
```

### Affected Resource(s)
- aws_emr_cluster

### Terraform Configuration Files
Example gist taken from the docs, including `instance_group` definition in #1071:
[EMR Cluster example with instance_group](https://gist.github.com/kh360/f0226dc2db2ebb1a29256a4467c2e484)

### Debug Output
```
$ terraform init

Initializing the backend...

Successfully configured the backend "s3"! Terraform will automatically
use this backend unless the backend configuration changes.

Initializing provider plugins...
- Checking for available provider plugins on https://releases.hashicorp.com...
- Downloading plugin for provider "aws" (0.1.4)...

Terraform has been successfully initialized!

$ terraform plan
2 error(s) occurred:

* aws_emr_cluster.emr-test-cluster: "master_instance_type": required field is not set
* aws_emr_cluster.emr-test-cluster: : invalid or unknown key: instance_group
```

### Expected Behavior
Should have provisioned a cluster with multiple MASTER and CORE nodes.

### Actual Behavior
Complained about not recognizing the `instance_group` key, even though that feature should be available in `v0.1.4`.

### Steps to Reproduce
1. Copy example gist to its own directory and configure S3 bucket for one with write access
2. `terraform init`
3. `terraform plan`

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1071

**Comment by e-carlin** [comment]
I ended up resolving the problem myself. The update to the Terraform AWS Provider was release on 9/27. I hadn't run terraform init since then so I had an outdated version of the provider. Deleting the .terraform directory and running terraform init resolved it.

@kh360  ^

**Comment by bflad** [comment]
This was previously released in [version 1.0.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#100-september-27-2017) and has been available in all releases since. Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
